cli/tests/integrations/test_task.py

```diff
@@ -68,50 +88,37 @@ def test_task():
 
     tasks = json.loads(stdout.decode('utf-8'))
     assert isinstance(tasks, collections.Sequence)
-    assert len(tasks) == 1
+    assert len(tasks) == NUM_TASKS
 
     schema = create_schema(task_fixture().dict())
     for task in tasks:
         assert not util.validate_json(task, schema)
 
-    _uninstall_sleep()
-
 
 def test_task_table():
-    _install_sleep_task()
-    assert_lines(['dcos', 'task'], 2)
-    _uninstall_sleep()
+    assert_lines(['dcos', 'task'], NUM_TASKS+1)
 
 
 def test_task_completed():
-    _install_sleep_task()
-    _uninstall_sleep()
-    _install_sleep_task()
-
     returncode, stdout, stderr = exec_command(
         ['dcos', 'task', '--completed', '--json'])
     assert returncode == 0
     assert stderr == b''
-    assert len(json.loads(stdout.decode('utf-8'))) > 1
+    assert len(json.loads(stdout.decode('utf-8'))) > NUM_TASKS
 
     returncode, stdout, stderr = exec_command(
         ['dcos', 'task', '--json'])
     assert returncode == 0
     assert stderr == b''
-    assert len(json.loads(stdout.decode('utf-8'))) == 1
-
-    _uninstall_sleep()
+    assert len(json.loads(stdout.decode('utf-8'))) == NUM_TASKS
 
 
 def test_task_none():
-    assert_command(['dcos', 'task', '--json'],
+    assert_command(['dcos', 'task', 'bogus', '--json'],
```

Comment:
Hmm. It would be nice if we could still test `dcos task --json` with on a DCOS cluster without any tasks. This, in theory, is a different code path, no?